code/modules/clothing/gloves/xeno/tajara.dm

```diff
@@ -0,0 +1,47 @@
+/obj/item/clothing/gloves/yellow/specialt
+	name = "tajaran electrical gloves"
+	desc = "These gloves will protect the wearer from electric shock. Made special for Tajaran use."
+	species_restricted = list("Tajara")
+
+/obj/item/clothing/gloves/black/tajara
+	desc = "Black gloves made for Tajara use."
+	species_restricted = list("Tajara")
+
+/obj/item/clothing/gloves/red/tajara
+	desc = "Red gloves made for Tajaran use."
+	species_restricted = list("Tajara")
+
+/obj/item/clothing/gloves/blue/tajara
+ 	desc = "Blue gloves made for Tajaran use."
+ 	species_restricted = list("Tajara")
+
+/obj/item/clothing/gloves/orange/tajara
+ 	desc = "Orange gloves made for Tajaran use."
+ 	species_restricted = list("Tajara")
+
+/obj/item/clothing/gloves/purple/tajara
+ 	desc = "Purple gloves made for Tajaran use."
+ 	species_restricted = list("Tajara")
+
+/obj/item/clothing/gloves/brown/tajara
+ 	desc = "Brown gloves made for Tajaran use."
+ 	species_restricted = list("Tajara")
+
+/obj/item/clothing/gloves/green/tajara
+ 	desc = "Green gloves made for Tajaran use."
+ 	species_restricted = list("Tajara")
+
+/obj/item/clothing/gloves/white/tajara
+ 	desc = "White gloves made for Tajaran use."
+ 	species_restricted = list ("Tajara")
+
+/obj/item/clothing/gloves/black/tajara/smithgloves
+	name = "machinist gloves"
+	desc = "Protective leather gloves worn by Adhomian urban workers."
```

Comment:
Ahdominian*

Reply:
It is Adhomai in auroraverse.

Reply:
Yep, just trying to wrap my head around the nomenclature there. Ahdomai > Ahdomian, or would it be Ahdomaian? Seems fine though